libraries/freertos_plus/aws/ota/src/aws_iot_ota_agent.c

```diff
@@ -1075,7 +1109,12 @@ static OTA_Err_t prvProcessDataHandler( OTA_EventData_t * pxEventData )
         if( xResult == eIngest_Result_FileComplete )
         {
             /* File receive is complete and authenticated. Update the job status with the self_test ready identifier. */
-            xOTA_ControlInterface.prvUpdateJobStatus( &xOTA_Agent, eJobStatus_InProgress, ( int32_t ) eJobReason_SigCheckPassed, ( int32_t ) NULL );
+            xErr = xOTA_ControlInterface.prvUpdateJobStatus( &xOTA_Agent, eJobStatus_InProgress, ( int32_t ) eJobReason_SigCheckPassed, 0 );
```

Comment:
Is type cast to `int32_t` required here ?

Reply:
yeah, seems redundant, will remove.